dist/gcc-compatible/EverCrypt_HMAC.h

```diff
@@ -38,11 +38,13 @@ extern "C" {
 
 
 #include "Hacl_Kremlib.h"
+#include "Hacl_Hash_SHA1.h"
 #include "Hacl_HMAC.h"
 #include "Hacl_Impl_Blake2_Constants.h"
-#include "Hacl_Hash.h"
+#include "Hacl_Hash_SHA2.h"
```

Comment:
Unrelated to this PR but just some food for thought.
I think we'll want to allow selecting algorithms for things like hmac as well in future. This will require some conditionals here on the includes as well as the functions.